src/runtime_src/core/pcie/driver/linux/xocl/subdev/ulite.c

```diff
@@ -339,7 +339,7 @@ static void ulite_shutdown(struct uart_port *port)
 }
 
 static void ulite_set_termios(struct uart_port *port, struct ktermios *termios,
-			      struct ktermios *old)
+			      const struct ktermios *old)
```

Comment:
Please avoid mixing tabs and spaces.
https://addons.mozilla.org/en-US/firefox/addon/refined-github-/ is showing this to me.

Reply:
Hi @keryell Thanks for the review. I have fixed it, please review and let me know.  Thank you.